### PR TITLE
doc: add onboarding step for pkgjs org

### PR DESCRIPTION
**Pull request by mhdawson**
Refs: https://github.com/nodejs/admin/issues/470

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
Sent invites to all existing TSC members who were not already org owners of the pkgjs org.

File: ONBOARDING.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ invalid and are superseded by the TSC Charter.
 * Change the new member's role in the GitHub `nodejs` organization to `Owner`.
 * Add them to the `@nodejs-private` org and with an `Owner` role.
 * Add them to the `@nodejs-private/security-tsc` team.
+* Add them to the `@pkgjs` org and with an `Owner` role.
```
Comment (ljharb) [inline ONBOARDING.md:17]: it might be good to add them to a "tsc" team on the pkgjs org, so it's clear *why* they're in the org?

**Review by gireeshpunathil — APPROVED** [review]

**Comment by Trott** [comment]
I don't think we should be adding people without adding them to a team that indicates why they're there. Otherwise, offboarding will be super error-prone. And it also means that people may be confused about what they are permitted to do. (As a TSC member, I'm probably given pretty broad authority. As a Moderation member, my authority is much more narrow. But if I and others don't know why I'm there--TSC member? Moderation member? Both?--it can lead to confusion.)

**Comment by mhdawson** [comment]
@Trott I had started with adding to a team, but since they also had to be added to the org, I ended up not doing that. Happy to have a team as well though if that helps clarify things.

File: ONBOARDING.md (hunk 1 above)
Comment (mhdawson) [inline ONBOARDING.md:17]:
```suggestion
* Add them to the `@pkgjs` org and with an `Owner` role.
* Add them to the `@pkgjs/node-tsc` team.
```

File: ONBOARDING.md [hunk 2]
```diff
@@ -28,6 +29,8 @@ invalid and are superseded by the TSC Charter.
 * Remove them from the `@nodejs-private/security-tsc` team.
 * Remove them from the `@nodejs-private` org unless they are members for a
   reason other than TSC membership.
+* Remove them from the `@pkgjs` org unless they are members for a
```
Comment (mhdawson) [inline ONBOARDING.md:32]: * Remove them from the `@pkgjs/node-tsc` team
* Remove them from the `@pkgjs` org unless they are members for a

File: ONBOARDING.md (hunk 1 above)
Comment (mhdawson) [reply ONBOARDING.md:17]: @ljharb , @Trott updated.

**Comment by mhdawson** [comment]
@ljharb @Trott look good to you now?

**Comment by ljharb** [comment]
LGTM

**Review by Trott — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Comment by mhdawson** [comment]
TSC members added in line with updated doc, landing.

**Pull request merged**